### DISCUSSION TITLE
How to skip the download for accelearte my scan job

### DISCUSSION THREAD

**Discussion by haojingcn** [aquasecurity/trivy#6932] [Q&A]
### Question

I meet this scan problem. EveryTime it will download some thing, how cloud it skip it, because i want use the cache?

![image](https://github.com/aquasecurity/trivy/assets/3139872/1e15e69f-4052-4b3b-b77a-44354ab24b76)

May be skip fanal???

### Target

Kubernetes

### Scanner

None

### Output Format

None

### Mode

Standalone

### Operating System

Linux

### Version

```bash
Version: 0.52.1
Vulnerability DB:
  Version: 2
  UpdatedAt: 2024-06-14 00:17:40.296054732 +0000 UTC
  NextUpdate: 2024-06-14 06:17:40.296054451 +0000 UTC
  DownloadedAt: 2024-06-14 01:54:49.080758186 +0000 UTC
Java DB:
  Version: 1
  UpdatedAt: 2024-06-13 01:03:26.782257844 +0000 UTC
  NextUpdate: 2024-06-16 01:03:26.782257723 +0000 UTC
  DownloadedAt: 2024-06-13 08:32:36.405146706 +0000 UTC
Check Bundle:
  Digest: sha256:cfb65621a1f55d9d099c4c28931b252716fcda8bba5081eb43f1001668e79d85
  DownloadedAt: 2024-06-13 08:30:39.474959836 +0000 UTC
```

**Comment by DmitriyLewen** [comment]
Hello @haojingcn 
Thanks for your report!

IIUC this progress bar shows checking of artifacts  (in your case Trivy detects 937 artifacts and checks them).
@chen-keinan right?

But perhaps we can add log message about this progress bar to avoid confusion.
@chen-keinan wdyt?

**Reply by chen-keinan** [reply]
Yes correct, it check the num of resource appear in progress bar

**Reply by haojingcn** [reply]
![image](https://github.com/aquasecurity/trivy/assets/3139872/99fdfc07-fa68-487b-9a99-69b668005930)

so this artifacts means the k8s resources. I got it, thx.

Can i make the pr for log message enhancement?

**Reply by chen-keinan** [reply]
@haojingcn sure go for it if you have time

**Reply by haojingcn** [reply]
I can't create an Issue myself because it would break the spec. Can you create an issue for me?

**Reply by haojingcn** [reply]
It's simple, so I just make the pr firstly. #6997
